src/fastcs/backend.py

```diff
@@ -155,3 +160,38 @@ async def scan_coro() -> None:
             await asyncio.gather(*[method() for method in methods])
 
     return scan_coro
+
+
+def build_controller_api(controller: Controller) -> ControllerAPI:
```

Comment:
Still think it would be nice for this should be a static `ControllerAPI` method, but up to you.

Reply:
I think I prefer it like this just because it means `Controller` and `ControllerAPI` have no knowledge of each other.